change-notes/1.19/analysis-javascript.md

```diff
@@ -2,7 +2,7 @@
 
 ## General improvements
 
-* Modelling of taint flow through array operations has been improved. This may give additional results for the security queries.
+* Modelling of taint flow through array and buffer operations has been improved. This may give additional results for the security queries.
```

Comment:
I would use "Modeling" as I believe it's the most common used form in the US. However, I leave this up to you as this spelling may be used elsewhere